packages/addons/service/nextpvr/package.mk

```diff
@@ -2,9 +2,9 @@
 # Copyright (C) 2021-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="nextpvr"
-PKG_VERSION="7.0.1~Omega"
-PKG_ADDON_VERSION="7.0.1~2"
-PKG_REV="2"
+PKG_VERSION="7.0.4~Omega"
+PKG_ADDON_VERSION="7.0.4~3"
```

Comment:
Please apply the same PKG_VERSION changes like you did in master

Reply:
I understand your reasoning , but I just tested the LE12 zip install on 3 platforms and I don't feel like doing it over but I will not release untested changes with different versions of shell script, python and Kodi  behaviour.  The non-house style was accepted before in LE11 and in LE12 and I had no issues changing for LE13 given it is still not a first release.

Reply:
@HiassofT turns out I figured a much better way to run my script and so I need to retest anyway and I will make the requested version change then to. 

I don't know how to postpone PR 10625 release hopefully is that something you or @CvH and I can create a new PR with changes I am testing now.